src/psyclone/gocean1p0.py

```diff
@@ -525,9 +557,17 @@ def _lower_bound(self):
             if go_kernels:
                 index_offset = go_kernels[0].index_offset
 
+            if self._loop_type == "inner":
```

Comment:
Is this part going to be OK in the presence of a GOLoopSwap transformation? I guess it is because I've had a quick look at the code for that transformation and AFAICS it doesn't modify  the loop_type of the loops being transformed. Is that correct?

Reply:
Yes, it just changes the tree, not the content of any of the nodes. So the former outer loop is after the transform the inner loop, but it is still marked as 'outer' loop (maybe instead of inner and outer, we should call it dim1 and dim2 or so ;) ). Besides, the swap transformation tests would have detected this.